doc/lib.rst

```diff
@@ -45,6 +50,8 @@ Automatic imports
   Nim message passing support for threads. **Note:** This is part of the
   system module. Do not import it explicitly. Enabled with `--threads:on`:option:.
 
+* `std/channels <channels.html>`_
```

Comment:
```suggestion
* `channels <channels.html>`_
```
Having an `std/` prefix should be the default anyway, all the other `std/`-only modules don't use it anymore either, since the module docs still show it.

Reply:
Okay, I see that, and just didn't realize that the documentation is in a bit of a state of flux with the `std/` prefixes being removed.  The minor problem is then that these new `std/` modules can only be imported using the new `import std/<module_name>`, but as you say as long as the `std/` module docs make it clear they are in this category, everything should be fine along with my shorted suggestion above.

Reply:
> The minor problem is then that these new `std/` modules can only be imported using the new `import std/<module_name>`

It's not a problem if everyone just uses the `std/` prefix for every stdlib module.

> as long as the `std/` module docs make it clear they are in this category, everything should be fine along with my shorted suggestion above.

To be clear, they don't distinguish between modules in the `std/` folder and other modules, they show the `std/` prefix for every stdlib module (with a few exceptions), because it's the recommended way to import them.